tools/autograd/templates/Functions.h

```diff
@@ -52,7 +52,6 @@ struct TypeAndSize {
 
   Tensor zeros() { return at::zeros_symint(sym_sizes, options); }
 
-private:
```

Comment:
is it nontrivial to utilize accessors for these from the codegen code or just more convenient this way?